### ISSUE TITLE
5.18.5.f Does not work on Rocky Linux 9.4

### ISSUE THREAD

**Issue by TreeBranches** [xavery/transgui#99]
Going by the error I'm getting, it looks like your fork of Transmission Remote GUI requires glibc-2.35

```
/tmp/.mount_transgyU4jbJ/AppRun.wrapped: /lib64/libm.so.6: version `GLIBC_2.35' not found (required by /tmp/.mount_transgyU4jbJ/usr/bin/../lib/libQt5Core.so.5)
```

After searching around it looks like Rocky Linux only currently supports up to v2.34. Is there a reason why 2.35 was chosen as a dependency for this fork? Please consider a lower version if not so important.

**Comment by lighterowl** [comment]
Hi there, thanks for the report.

There is no particular reason and the dependency is just a consequence of building under Ubuntu 22.04. Switching to 20.04 would fix your problem as it uses glibc 2.31 but I'll look into switching to Rocky/Alma 8 instead so the generated binary can have the widest compatibility.

**Comment by lighterowl** [comment]
This should be fixed now, please try out the build at https://github.com/lighterowl/transgui/actions/runs/10495582943/artifacts/1838926329 .

**Comment by lighterowl** [comment]
Tested with Rocky 8.10 and Rocky 9.4, closing as fixed in 5.18.6.
